packages/pubsub/src/v1/publisher_client.js

```diff
@@ -827,10 +855,11 @@ function PublisherClientBuilder(gaxGrpc) {
   }]);
   extend(this, publisherClient.google.pubsub.v1);
 
-  var grpcClients = {
-    iamPolicyClient: iamPolicyClient,
-    publisherClient: publisherClient
-  };
+  var grpcClients = merge(
```

Comment:
You should get the same results with `extend(true, {}, iamPolicyClient, publisherClient)`.